vocabularies-qsi/addr-geocode-types.ttl

```diff
@@ -11,203 +11,153 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 :building-access-point
     a skos:Concept ;
-    dcterms:provenance "AS 4590.1:2017 | Standards Australia" ;
```

Comment:
I assumed this information would be provenance rather than history note?

Reply:
The latest version of VocPub used `skos:historyNote` instead of `dcterms:provenance`.

See the version change info here https://agldwg.github.io/vocpub-profile/specification.html.